.github/ISSUE_TEMPLATE/documentation.md

```diff
@@ -0,0 +1,18 @@
+---
+name: Documentation
+about: Suggest an improvement for the documentation
+title: "[DOCUMENTATION]"
+labels: documentation
+assignees: ''
+
+---
+Please use either use option A) or B) and fill in the selected options:
+
+**A. I want to add something to this section:**
```

Comment:
I would state that they have to paste a link to the section?

Reply:
added